### PR TITLE
Changes to GitNotes.txt

### PR DESCRIPTION
**Pull request by Elrynthir**
I have added a few definitions to our GitNotes File.

### CODE REVIEW

**Comment by Kaanin25** [comment]
Can you fix the tabbing so it lines up?

**Comment by Elrynthir** [comment]
Tabbing is fixed for the web view of GitNotes.  
>.<

File: GitNotes.txt [hunk 1]
```diff
@@ -3,33 +3,43 @@ GitNotes
...
-	git remote prune origin		removes stale branches from local view
+	git remote prune origin			removes stale branches from local view
+
+	git remote prune origin --dry-run	returns a list of branches that would be pruned
+
+	git fetch --all				pulls the data from your current branch without merging
+
+	git reset				resets the master to what was fetched
+
+	git reset --hard origin/master		swaps your working copy with the master branch deleting any current saved/committed work
```
Comment (Kaanin25) [inline GitNotes.txt]: Can you please make the description of this one shorter? Thanks

Comment (Kaanin25) [inline GitNotes.txt]: Can you please make the description on this one shorter? Thanks!

Comment (Elrynthir) [inline GitNotes.txt]: Fixed.

**Pull request merged**